salve_ipc/ipc.py

```diff
@@ -16,12 +16,6 @@
 )
 from .server import Server
 
-# Deal with Windows weirdness
-if platform == "win32":
```

Comment:
Not directly related to tree sitter but I realized the connection was an unused variable so I've removed them